js/analyzer.js

```diff
@@ -24,11 +24,14 @@ Analyzer.prototype.init = function() {
     if(typeof this.config.queue !== 'undefined' && this.config.queue.length !== 0){
         this.queue = this.config.queue;
     }else{
-        this.queue = ['keywordDensity', 'subheaderChecker'];
+        this.queue = ['keywordDensity', 'subheaderChecker', 'stopwordChecker'];
     }
     //if no available keywords, load default array
-    if(typeof this.config.wordsToRemove === 'undefined'){
-        this.config.wordsToRemove =  [' a', ' in', ' an', ' on', ' for', ' the', ' and'];
+    if(typeof this.config.wordsToRemove == 'undefined'){
+        this.config.wordsToRemove =  analyzerConfig.wordsToRemove;
+    }
+    if(typeof this.config.stopWords == 'undefined'){
+        this.config.stopWords = analyzerConfig.stopWords;
     }
```

Comment:
This is changed in https://github.com/Yoast/js-text-analysis/tree/stories/DT/fleschReading. The init function is split up in multiple functions to avoid clutter.